README.md

```diff
@@ -312,6 +312,24 @@ algorithm](https://nodejs.org/api/modules.html#modules_require_resolve_request_o
 the correct `node_modules` directories are updated. With this argument set, all locally specified
 target dependencies are also packed into the working directory's `node_modules` directory.
 
+## Troubleshooting
+
+### Npm-pack-here runs but webpack (or other bundler) doesn’t detect any changes
+
+As of Webpack v5 the default `node_modules` caching settings will likely cause issues when attempting
+to do local testing with `npm-pack-here`. The problem is the
+[`snapshot.managedPaths`](https://webpack.js.org/configuration/other-options/#managedpaths) default
+value. It appears like webpack’s caching does not detect any changes to `file:` dependencies as different.
+To resolve this, remove the `node_modules` default value`snapshot.managedPaths = []`.
```

Comment:
This is confusing.  Are you saying add `snapshot.managedPaths = []` to your config?  Or remove this line from your config?

Reply:
add, if its not set to empty then it picks up the default which is `snapshot.managedPaths = [path.resolve(__dirname, '../node_modules')]`

Reply:
Can you say something like 

To resolve this, add `snapshot.managedPaths = []` to your configuration to override the default value (`node_modules`).

Reply:
what do you think of the changes?

Reply:
lgtm!